dojo/models.py

```diff
@@ -1413,101 +1413,288 @@ class Finding(models.Model):
-    sast_source_object = models.CharField(null=True, blank=True, max_length=500, help_text="Source object (variable, function...) of the attack vector")
-    sast_sink_object = models.CharField(null=True, blank=True, max_length=500, help_text="Sink object (variable, function...) of the attack vector")
-    sast_source_line = models.IntegerField(null=True, blank=True,
+    sourcefilepath = models.TextField(null=True,
+                                      blank=True,
+                                      editable=False,
+                                      verbose_name="Source File Path",
+                                      help_text="Filepath of the source code file in which the flaw is located.")  # Not used? to remove
+    sourcefile = models.TextField(null=True,
+                                  blank=True,
+                                  editable=False,
+                                  verbose_name="Source File",
+                                  help_text="Name of the source code file in which the flaw is located.")
+    param = models.TextField(null=True,
+                             blank=True,
+                             editable=False,
+                             verbose_name="Parameter",
+                             help_text="Parameter used to trigger the issue (DAST).")
+    payload = models.TextField(null=True,
+                               blank=True,
+                               editable=False,
+                               verbose_name="Payload",
+                               help_text="Payload used to attack the service / application and trigger the bug / problem.")
+    hash_code = models.CharField(null=True,
+                                 blank=True,
+                                 editable=False,
+                                 max_length=64,
+                                 verbose_name="Hash Code",
+                                 help_text="A hash over a configureable set of fields that is used for findings deduplication.")
+    line = models.IntegerField(null=True,
+                               blank=True,
                                verbose_name="Line number",
-                               help_text="Source line number of the attack vector")
-    sast_source_file_path = models.CharField(null=True, blank=True, max_length=4000, help_text="Source filepath of the attack vector")
-    nb_occurences = models.IntegerField(null=True, blank=True,
-                               verbose_name="Number of occurences",
-                               help_text="Number of occurences in the source tool when several vulnerabilites were found and aggregated by the scanner")
+                               help_text="Line number. For SAST, when source (start of the attack vector) and sink (end of the attack vector) information are available, put sink information here.")
```

Comment:
This one is a bit weird

Reply:
I thought the original one was ok? (Note, I/tools I use never really used that field)

Reply:
I think the original is fine here too.

Reply:
Agree, it's a bad copy / paste from the documentation.